src/helpers/helpers.options.js

```diff
@@ -51,6 +52,19 @@ export function toPosition(value) {
   return readValueToProps(value, ['x', 'y'], 'center');
 }
 
+export function getTPosition(position, def = 0.5) {
```

Comment:
I think you can use getSize instead of this.

Reply:
getSize is implemented thinking about the size. getTPosition should return a value between 0 and 1.

Trying to use getSize, I should pass always 1 as size otherwise the calculation by string will not be correct. But if the string is not a percentage, it should return 0.5 (center, default) and not 0 (as getSize is doing) and therefore I should add another parameter to getSize with def.

Let try to change getSize and also toPercent enabling a default.

Reply:
Can you have a look now? Better? I left getTPosition as const invoking getSize